README.md

```diff
@@ -31,17 +34,35 @@ func init() {
 	logrus.SetOutput(os.Stderr)
 	logrus.SetLevel(logrus.DebugLevel)
 
+	// Log timestamps in UTC
+	timeString := ""
+
+	timestampLocale, err := time.LoadLocation("UTC") // e.g. "America/New_York"
+	if err != nil {
+		logrus.WithError(err).Debugf("Unable to determine timestampLocale, defaulting to local runtime")
+
+		timeString = time.Now().String()
+	} else {
+		logrus.WithField("timestampLocale", timestampLocale).Debugf("Found custom logging locality")
+
+		timeString = time.Now().In(timestampLocale).String()
+	}
+
+	// timeZoneTokens => [2017-12-23] [11:45:53.0703314] [-0000] [UTC]
+	timeZoneToken := strings.Split(timeString, " ")[3]
+
+	timeStampFormat := "Jan 2 15:04:05.00000 " + timeZoneToken
+
 	logrus.AddHook(discordrus.NewHook(
 		// Use environment variable for security reasons
 		os.Getenv("DISCORDRUS_WEBHOOK_URL"),
 		// Set minimum level to DebugLevel to receive all log entries
 		logrus.DebugLevel,
 		&discordrus.Opts{
-			Username: "Test Username",
-			Author: "", // Setting this to a non-empty string adds the author text to the message header
-			DisableTimestamp: false, // Setting this to true will disable timestamps from appearing in the footer
-			TimestampFormat: "Jan 2 15:04:05.00000", // The timestamp takes this format; if it is unset, it will take logrus' default format
-			EnableCustomColors: true, // If set to true, the below CustomLevelColors will apply
+			Username:            "",
```

Comment:
Please be sure to keep the values from the old Opts.

Reply:
✔